README.md

```diff
@@ -1,4 +1,15 @@
-# intern2021
+# Journey
+Journey is a learning experience running on [.NET Interactive Notebooks](https://github.com/dotnet/interactive) where teachers can design a step-by-step path for students. Along the student's journey, teachers can create challenges, provide customized feedback, and define rules to continue.
 
+## About Journey
+We believe that the journey towards a goal is one of the most crucial aspects of learning. As students, we wanted to create an engaging notebook experience for the classroom that utilized this idea. 
+Currently, Journey supports the following features:
+- Defining rules and challenges
+- Programatic revealing of cells
+- Customized teacher feedback with support for visual libraries
+- Re-route a student's path based on their answer
+- Access to student submission history
+
+## Get Started
+For information on configuring Journey on your computer, please refer to our [tutorial page](link).
```

Comment:
How does this sound?
To get started creating a learning journey, check out our tutorial!